lib/modules/dosomething/dosomething_gladiator/dosomething_gladiator.install

```diff
@@ -90,3 +92,20 @@ function dosomething_gladiator_update_7002(&$sandbox) {
     }
   }
 }
+
+/**
+ * Update response values to be a long text field, as the response is LONG.
+ */
+function dosomething_gladiator_update_7003(&$sandbox) {
+  $field = 'response_values';
+  $table = 'dosomething_gladiator_failed_task_log';
+  // Update the field to a large text.
+  db_change_field($table, $field, $field, array(
+    'description' => 'The JSON response from Gladiator',
+    'type' => 'text',
+    'length' => '500000',
+    'not null' => FALSE,
+    'default' => NULL,
```

Comment:
Noob question: why do you have to pass this array twice? You defined it above as `response_values` and then pass the same array into this function?